src/extension.ts

```diff
@@ -127,8 +127,6 @@ export async function activate(context: vscode.ExtensionContext) {
 		// Because auth provider implementation is in the same extension, we need to wait for it to activate first
 		sessionService.didFirstLoad.then(async () => {
 			if (remoteConnectionInfo) {
-				commandManager.register({ id: 'gitpod.api.autoTunnel', execute: () => remoteConnector.autoTunnelCommand });
```

Comment:
let's keep as a dummy thing here but clean up in https://github.com/gitpod-io/gitpod-code/blob/1de61b5711089287b7f27bc1223c57d6d9bb3144/gitpod-remote/src/extension.ts#L49